app/grandchallenge/cases/models.py

```diff
@@ -292,6 +309,24 @@ class Image(UUIDModel):
     color_space = models.CharField(
         max_length=5, blank=False, choices=COLOR_SPACES
     )
+    patient_id = models.CharField(max_length=64, null=True, blank=True)
+    patient_name = models.CharField(max_length=324, null=True, blank=True)
+    patient_birth_date = models.DateField(null=True, blank=True)
+    patient_age = models.CharField(max_length=4, null=True, blank=True)
+    patient_sex = models.CharField(
+        max_length=1,
+        null=True,
+        blank=True,
+        choices=PATIENT_SEX_CHOICES,
+        default=PATIENT_SEX_EMTPY,
```

Comment:
The value for `PATIENT_SEX_EMPTY` is `None`. I implemented this like the `stereoscopic_choice` field which I think is correct (unlike the `eye_choice` field).

Reply:
Then the default should be an empty string and `null` removed. This is handled automatically by Django, so really, the `PATIENT_SEX_EMTPY` is not necessary.

Reply:
Ah right. But the value does need to be defined in `choices` then, right? So then for clarity I would just keep it in here as well but change it to be `""` instead of `None`.

Reply:
It does not, see https://github.com/comic/grand-challenge.org/blob/122d60a8be7637df7f5b16b3520a7401b6f19e23/app/grandchallenge/workstation_configs/models.py#L164-L166 (although I think leaving out `default=""` only works as it was part of an initial migration, could be wrong).